exercises/matrix/matrix_test.rb

```diff
@@ -36,4 +36,10 @@ def test_extract_another_column
     matrix = Matrix.new("89 1903 3\n18 3 1\n9 4 800")
     assert_equal [1903, 3, 4], matrix.columns[1]
   end
+
+  def test_extract_last_column
+    skip
+    matrix = Matrix.new("1 2 3\n10 20 30\n100 200 300")
+    assert_equal [3, 30, 300], matrix.columns[2]
```

Comment:
We should be able to confirm (even if the data changes) that it is indeed the last.  So is it possible to do:

```suggestion
    assert_equal [3, 30, 300], matrix.columns.last
```

Rather than have to count or know that the last one is the last one?  Another variant (and common) way would be using the `-1` instead, such as:

```suggestion
    assert_equal [3, 30, 300], matrix.columns[-1]
```

Reply:
@kotp I think I the `.last` variant is most suitable here considering the test case name as `last_column`.